### ISSUE TITLE
Import com.anysoftkeyboard.languagepack.portuguese

### ISSUE THREAD

**Issue by menny** [AnySoftKeyboard/AnySoftKeyboard#2535]
and publish

**Comment by menny** [comment]
@somini, is https://github.com/AnySoftKeyboard/AnySoftKeyboard/pull/2590 the fix for the import?

**Comment by somini** [comment]
Yes, that's right. I imported that to the new repo and did some extra fixes on top.

As I mentioned elsewhere, feel free to publish this like the other language packs.
